thresher/admin.py

```diff
@@ -1,6 +1,21 @@
 from django.contrib import admin
 from thresher.models import *
```

Comment:
Avoid doing starred imports if possible to help code checkers and other developers parse the code more easily.  ``from thresher.models import (Client, ..., ...)``.